### DISCUSSION TITLE
F1 Watch Score | Is it worth it to rewatch a race I missed?

### DISCUSSION THREAD

**Discussion by racoon1210** [br-g/openf1#130] [Show and tell]
First of all thanks to openf1!
Two weeks ago I never touched anything regarding an API or Python and today I have a first rough version of what came to my mind some weeks ago.

**Do you also sometimes do not have the time to watch the race live?**
As a F1TV subscriber I am able to rewatch the race later and I always try not to get spoilered. Therefore I do not open instagram, any news app and not even LinkedIn, as I was spoilered even there once 😆 
Sometimes it is even hard to find the time then to rewatch the race, but I really like to do so, especially, when it was an interesting and exiting race! 

➡️ **But how should I know, if it is worth to rewatch without getting spoilered?**
Here comes my idea of a F1 Watch Score into play.
In the screenshot you can see the app i created for this until now, which captures the basics from what I thought of.
![image](https://github.com/user-attachments/assets/65c452c5-97f9-4f6b-9a02-c94ce2037188)

The details you can read in the textbox there are diabled per default and you only see the score first.
The challenge is still the score, as it has no limit and the calculation feels not that mature. But I will check this with the next races.
Interlagos this year e.g. has a score from over 100, which indicates for sure the excitement this race had 😆 🚀 

Do you have anything in mind, what should be added to a watch score like this or any other idea in this direction?
Would love to get some input in the comments from you, as it seems that we have way more experienced people around and also some creative minds!

**Comment by br-g** [comment]
Looks interesting! Thanks for sharing.
Please keep us posted when it's released :)

**Comment by CyrilNb** [comment]
this idea is brilliant!
I would love to implement something like to my [mac app]([apps.apple.com/app/gp-calendar-f1/id1632951751](http://apps.apple.com/app/gp-calendar-f1/id1632951751)) 😁

I don't know we can collaborate on this mate, your idea is really great

**Reply by racoon1210** [reply]
Would love to get some thoughts and ideas to improve it and also to finetune the way the score is calculated.

Don't know where it is the best way to collaborate here 😊

Btw if your link should lead to something, it is not working for me :/

**Reply by CyrilNb** [reply]
> Btw if your link should lead to something

strange, here it [GP Calendar Mac](http://apps.apple.com/app/gp-calendar-f1/id1632951751)
